src/wasm-interpreter.h

```diff
@@ -455,7 +455,14 @@ class ModuleInstance {
             case TruncSFloat64:    return truncSFloat(curr, value);
             case TruncUFloat64:    return truncUFloat(curr, value);
             case ReinterpretFloat: return value.castToI64();
-            case DemoteFloat64:    return value.truncateToF32();
+            case DemoteFloat64:    {
```

Comment:
please only one space between `:` and `{`

otherwise lgtm

btw, if it's faster for you to do one PR with multiple small changes like this, i'm fine either way